### ISSUE TITLE
Update Date formatting in process models (instructions)

### ISSUE THREAD

**Issue by calexh-sar** [sartography/spiff-arena#698]
Update Date formatting in process models (instructions) so that it’s aligned with the system format  e.i. 23-Oct-2023.

There is a new mechanism that could be used to accomplish this that has recently been developed. SPIFF_FORMAT:::. it is the greatest feature we have ever developed. ask KB and Jason.

(Created from https://www.notion.so/Update-Date-formatting-in-process-models-instructions-1f0f69a6d10942c8a8a39460f4ad6b0b)

**Comment by calexh-sar** [comment]
SPIFF_FORMAT:::. examples KB mentioned:
- [Example in use](https://github.com/sartography/sartography-process-models/blob/main/extensions/my-time-tracking-report/results-markdown.md#L1) (click on the `Raw` button for best view)
- [All available options](https://github.com/sartography/spiff-arena/blob/main/spiffworkflow-frontend/src/services/FormattingService.tsx#L4)

**Comment by calexh-sar** [comment]
@sashayar13 discussed using the SPIFF_FORMAT::: script with Jason and KB and this seems to be the best way to handle this ticket, but the work in #637 would need to be completed first.  Since this is a low priority ticket, I am going to put this ticket back in the backlog and put a comment on #637 to revisit once that ticket is complete.
